docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1420,6 +1422,7 @@ must restart the server for that change to take effect.
 |`fhirServer/core/defaultHandling`|Y|Y|
 |`fhirServer/core/allowClientHandlingPref`|Y|Y|
 |`fhirServer/core/checkReferenceTypes`|N|N|
+|`fhirServer/core/conditionalDeleteMaxNumber`|N|N|
```

Comment:
because you use FHIRConfigHelper to look up the value of the config property, I think this means it *is* client-specific and dynamic
```suggestion
|`fhirServer/core/conditionalDeleteMaxNumber`|Y|Y|
```

Reply:
good catch, actually I was not sure about this.

Reply:
done